tdrs-backend/tdpservice/parsers/schema_defs/tanf/t2.py

```diff
@@ -112,7 +112,7 @@
                     condition_function=validators.oneOf((1, 2)),
                     result_field="WORK_PART_STATUS",
                     result_function=validators.oneOf(
-                        ["01", "02", "05", "07", "09", "15", "17", "18", "19", "99"]
+                        ["01", "02", "05", "07", "09", "15", "17", "18", "19", "  "]
```

Comment:
i might be overthinking this, but... `WORK_PART_STATUS` is an optional field, so blanks are handled within `parse_and_validate`. i wonder if `if_then_validator` should do something similar, taking into account the `required` attribute of the field so blanks (or potentially other null/empty characters) don't explicitly need to be added to the postparsing validators

Reply:
I think we could definitely do that and doing so prevents errors where we forgot to include the blank. Does that creep on scope for this ticket though? Im happy to implement here as long as no one objects.

Reply:
captured in #2821